packages/gestalt-design-tokens/tokens/vr-theme-web-mapping/comp-web-color-light.json

```diff
@@ -259,6 +259,19 @@
           }
         }
       },
+      "spinner": {
```

Comment:
you dont need this either

Reply:
+1 - the VR tokens will be pulled in here